### ISSUE TITLE
Lack of documentation for smoothly changing volume using interactive float

### ISSUE THREAD

**Issue by Keyne** [savonet/liquidsoap#597]
There's no docs or perhaps the functionality itself about how to drop or rise gain smoothly, not suddenly using an interactive float through telnet.
I know that the mail list is the right place for asking this first but I'm not approved to post there.

Does liquidsoap has this function? Does the docs shows this information? If so where?

**Comment by toots** [comment]
http://www.liquidsoap.info/doc-1.3.3/metadata.html :-)

**Comment by toots** [comment]
Whoops wrong issue sorry!

**Comment by toots** [comment]
No, there isn't for now. It's not an easy task actually. You can look at `smooth_add` in `utils.liq` for a similar example. If you intend to do it on an ongoing stream I'll have to think about it, perhaps an option or new operator would be needed.

**Comment by toots** [comment]
Ok, thanks for suggesting this. I've been looking at it and came up with some nice new features!

This might need some documentation to be properly explained but you might want to look at https://github.com/savonet/liquidsoap/pull/664 where the fade operators are re-implemented on the script side!

I'm gonna let this sit for a minute to make sure it doesn't have any issue I didn't see but I'll probably merge it soon..
